style.css

```diff
@@ -859,4 +949,4 @@ input:focus:invalid {
   #form-icons img:hover {
     cursor: pointer;
   }
-}
+}
```

Comment:
As stated by the previous reviewer, your project container still doesn't fit the view width. kindly fix this problem do as to have a better UI

Figma design
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/20537712/f6f71468-077f-45bc-ac69-1b73cf55dab8)

your design :
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/20537712/40dbc70b-8a68-4e03-8b84-4c8b685fcd2a)

Reply:
@Tshepo45 
please kindly implement the last changes I suggested before submitting for review. your project container still doesn't fit the view width. kindly fix this problem do as to have a better UI 

your design :
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/20537712/40dbc70b-8a68-4e03-8b84-4c8b685fcd2a)